### PR TITLE
Sync ex versioning

### PR DESCRIPTION
**Pull request by benghaziboy**
@mparent61 @MichaelButkovic

### CODE REVIEW

**Comment by MichaelButkovic** [comment]
:+1:

File: djes/management/commands/sync_es.py [hunk 1]
```diff
@@ -35,10 +35,14 @@ def get_indexes():
 def get_latest_index_version(name):
     conn = connections.get_connection('default')
     alias = conn.indices.get_alias(name)
+    if not alias:
```
Comment (mparent61) [inline djes/management/commands/sync_es.py]: non-existent alias raised an exception in Travis:  

`NotFoundError: TransportError(404, u'IndexMissingException[[avclub-integration] missing]')`

https://magnum.travis-ci.com/theonion/avclub/builds/16386430

**Comment by mparent61** [comment]
LGTM

**Comment by benghaziboy** [comment]
@mparent61

**Comment by mparent61** [comment]
:+1:

**Pull request merged**